bigbluebutton-html5/imports/ui/components/subscriptions/component.jsx

```diff
@@ -164,6 +167,10 @@ export default withTracker(() => {
       },
       ...subscriptionErrorHandler,
     });
+
+    Object.values(localCollectionRegistry).forEach(
```

Comment:
Also add a throttle here, if it runs as is, it'll generate 961 calls for `checkForStaleData`

Reply:
If it generates 961 calls, throttling does not seem like the solution - if anything it should be debounced.
Do you have a different place to suggest where this will be called only when absolutely necessary?

Reply:
Also: how did the 961 calls number come up here? Can you provide a reproducible scenario so I can tinker with it?

The way I initially came up with this code is hat this loop would only be run once every subscription handler is ready - so this would amount to the N collection calls to checkForStaleData (which currently is 32).

I just re-validated that adding a call counter stemming from this entrypoint only and it's just that: 32 calls per (re)-connection.